src/interceptor/utils/isIgnoredRequest.ts

```diff
@@ -0,0 +1,39 @@
+const commonLocalUrlTlds = [
+  'local'
+]
+
+export function isAnIgnoredRequest ({ url, ignoredDomains, baseUrl, allowLocalUrls }: {
+  url: URL, ignoredDomains: string[], baseUrl: string, allowLocalUrls: boolean
+}): boolean {
+
+  const { origin: baseOrigin } = new URL(baseUrl);
+  const hostname = url.hostname;
+
+  // Ignore intercepting responses to supergood
+  if(baseOrigin === url.origin) {
+    return true;
+  }
+
+  if(!hostname && !allowLocalUrls) {
```

Comment:
is this case possible?

Reply:
I am not 100% sure, I was trying to be conservative.